tools/run_tests/xds_k8s_test_driver/framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -98,6 +98,17 @@ def stop_pod_dependencies(self, *, log_drain_sec: int = 0):
 
         self.pod_log_collectors = []
 
+    def get_pod_restarts(self, deployment: k8s.V1Deployment) -> int:
```

Comment:
If you want - I'm ok with overriding this at the client/server level, and just calling super().get_pod_restarts(self.deployment)

Reply:
IMO it's fine to keep it since it's not that complicate.